deploy/test-environments/README.md

```diff
@@ -101,4 +108,23 @@ terraform apply --auto-approve -target "module.ec_deployment"
 
 ```bash
 terraform apply --auto-approve -target "module.eks"
+```
+
+## Environment Cleanup
+
+To destroy local environment use
+
+``` bash
+terraform destroy -var="region=eu-west-1"
+```
+
+
+To destroy the environment provisioned using the Sanity job, follow these steps:
+
+1. [Download](https://s3.console.aws.amazon.com/s3/buckets/tf-state-bucket-test-infra?region=eu-west-3&tab=objects) the Terraform state file to the [test-environments](../test-environments/) folder.
+2. Rename the state file, for example, `terraform-sanity.tfstate`.
```

Comment:
why do we need to rename them

Reply:
We need to rename the file to prevent conflicts with the `terraform.tfstate` file. In case you are running Terraform locally, there is a possibility that the file already exists. Renaming it helps to avoid conflicts or accidental deletion